packages/tldraw/src/lib/shapes/text/TextShapeUtil.tsx

```diff
@@ -65,6 +79,21 @@ export class TextShapeUtil extends ShapeUtil<TLTextShape> {
 		})
 	}
 
+	override getFontFaces(shape: TLTextShape): TLFontFace[] {
+		const list = getFontsFromRichText(
+			shape.props.richText,
+			tipTapDefaultExtensions,
+			{
+				family: `tldraw_${shape.props.font}`,
+				weight: '500',
```

Comment:
i see 500 in lots of places as the default. our current base font weight is actually 400. wdyt?